portfolio/src/main/java/com/google/sps/servlets/PersonalityDataServlet.java

```diff
@@ -23,16 +23,32 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
 
 
 @WebServlet("/personality-data")
 public class PersonalityDataServlet extends HttpServlet {
 
-  private Map<String, Integer> personalityTypeVotes = new HashMap<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Make a class variable so you can use the same instance throughout all these methods